### ISSUE TITLE
Lab 06 Part 1

### ISSUE THREAD

**Issue by dshartmann** [DS4PS/cpp-523-spr-2021#29]
Ok I am a little embarrassed to be posting this because I am sure I am just overlooking something obvious...but I keep getting this error when I try to run the chunk for the table in the first question. I'm sure I am supposed to input some data somewhere but I have been playing with it for a while and just can't seem to figure it out. 
<img width="667" alt="Screen Shot 2021-02-19 at 1 08 55 PM" src="https://user-images.githubusercontent.com/77824972/108584219-cc4f3480-72fc-11eb-851c-1cfec9a5a4ad.png">
 advice!

**Comment by lecy** [comment]
Look a few lines up. You should be creating the variables before this chunk, something like:

    X <- income
    Y <- happiness

**Comment by dshartmann** [comment]
So am I creating a new chunk to run creating the variables as you described? I tried this, and also tried copying the code from the lab but it still doesn't seem to be right.

**Comment by lecy** [comment]
I think you just need to rename the variables. 

```r
# divide income by 10,000
y.happy <- dat$happiness
x.income <- dat$income / 10000  
x2.income.sq <- x.income*x.income
summary( lm( y.happy ~ x.income + x2.income.sq ) )
```
Note no data=dat in the lm() function.

**Comment by AmyTrumpower** [comment]
I had the same issue but had better luck following the code that is on the HTML page of the Lab 6 Specification (https://ds4ps.org/cpp-523-spr-2021/labs/lab-06-instructions.html) and not in the RTM file- they are slightly different. 

I'm unclear on the highlighted part b/c:
![image](https://user-images.githubusercontent.com/72533688/108600723-c1c98500-7366-11eb-9cd0-913080301b0f.png)

It instructs to set x equal to 15 and 25 but yesterday in the review we set it to 1.5 and 2.5; which makes more sense b/c we divided by 10,000.

**Comment by dshartmann** [comment]
Ok I think I got it!

<img width="520" alt="Screen Shot 2021-02-20 at 10 10 21 AM" src="https://user-images.githubusercontent.com/77824972/108603549-d35e5d00-7365-11eb-90f5-2f1376b476d1.png">

So for the rest of the lab, do we just create new chunks from the code on the HTML page? 

<img width="550" alt="Screen Shot 2021-02-20 at 10 30 57 AM" src="https://user-images.githubusercontent.com/77824972/108603836-651a9a00-7367-11eb-893a-2f2dd6e4a547.png">

<img width="665" alt="Screen Shot 2021-02-20 at 10 35 31 AM" src="https://user-images.githubusercontent.com/77824972/108603841-69df4e00-7367-11eb-8cf2-88f2f0f6495f.png">

**Comment by dshartmann** [comment]
> I had the same issue but had better luck following the code that is on the HTML page of the Lab 6 Specification (https://ds4ps.org/cpp-523-spr-2021/labs/lab-06-instructions.html) and not in the RTM file- they are slightly different.
> 
> I'm unclear on the highlighted part b/c:
> ![image](https://user-images.githubusercontent.com/72533688/108600723-c1c98500-7366-11eb-9cd0-913080301b0f.png)
> 
> It instructs to set x equal to 15 and 25 but yesterday in the review we set it to 1.5 and 2.5; which makes more sense b/c we divided by 10,000.

Yes, Amy, that makes more sense to me as well. I just changed it and the answer seems more accurate...

<img width="762" alt="Screen Shot 2021-02-20 at 10 44 14 AM" src="https://user-images.githubusercontent.com/77824972/108604027-9cd61180-7368-11eb-99e4-98a4f697f455.png">

**Comment by AmyTrumpower** [comment]
I got the same thing as you did on that one- so we are on the same track.

**Comment by dshartmann** [comment]
> I got the same thing as you did on that one- so we are on the same track.

Awesome! So I kept looking at it, and doesn't it make more sense for the variables 15 and 25 (or 1.5 and 2.5) to be changed to 15 and 10 (1.5 and 1) since those are the incomes we are working with in the equation? And shouldn't formula at the end to be adding the two together rather than subtracting them?
<img width="659" alt="Screen Shot 2021-02-20 at 10 49 00 AM" src="https://user-images.githubusercontent.com/77824972/108604573-060b5400-736c-11eb-8a09-736a906b58c9.png">
